### ISSUE TITLE
Disable key mapping helper preview window

### ISSUE THREAD

**Issue by petobens** [ranger/ranger#1263]
Hi! I've read the man pages but I cannot find how to the disable the popup window that shows the available mapping. In the following GIF when I press `,` there is a window showing all the posible `,<KEY>` combination. Can that window be disabled? Thanks in advance

![ranger](https://user-images.githubusercontent.com/2583971/44283380-9d8c0b00-a234-11e8-8f79-d20aaa8ec05d.gif)

**Comment by toonn** [comment]
Not currently possible. You might be interested in lowering the `hint_collapse_threshold` setting to minimize the list.

I'm considering having a threshold of 0 mean not drawing it at all.

Why exactly do you want to disable it though? it's a pretty useful discovery feature usually.

**Comment by petobens** [comment]
> Not currently possible. You might be interested in lowering the hint_collapse_threshold setting to minimize the list.

I tried `set hint_collapse_threshold 1` but the behaviour didn't change. Is that a bug? 

> Why exactly do you want to disable it though? it's a pretty useful discovery feature usually.

I find it distracting (I know the bindings by now).

**Comment by toonn** [comment]
Maybe you're misunderstanding what it does? Check the difference when pressing `+` with a `hint_collapse_threshold` of 30 and anything below 30.

**Comment by toonn** [comment]
No, as I said, it's just the closest you can get rn. A threshold of 0 doesn't seem significantly different from a threshold of 1 to me so I'm considering having it mean disable the hints altogether.

**Comment by petobens** [comment]
Ok. Can we leave this open then?

**Comment by toonn** [comment]
Yep. I'll ask for opinions. We could introduce a completely different setting too of course.

**Comment by sadid** [comment]
I use that menu sometimes but it would be cool if I could disable it by an option/state altogether.

**Comment by parmort** [comment]
Maybe adding a setting to white-/black-list certain prefix keys to pull up the preview window if it is enabled. I find that it was helpful first getting in to learn the commands and how they differ slightly from vim. But I do find it an annoyance sometimes if I know exactly what I want from a certain prefix key.

**Comment by toonn** [comment]
That's an interesting suggestion. I do wonder if it'd be used much though. I think most people either want them or not (excluding maybe bookmarks?) and having a toggle would make it easy to enable them briefly. With a list you might not remember what you blacklisted and add a keybinding and end up confused maybe? Either way it's harder to toggle.
